.github/workflows/_test_requiring_torch_cuda.yml

```diff
@@ -94,7 +94,8 @@ jobs:
           pip install -U --pre jaxlib -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html
           pip install -U --pre jax-cuda12-pjrt jax-cuda12-plugin -f https://storage.googleapis.com/jax-releases/jax_cuda_plugin_nightly_releases.html
           pip install -U --pre jax -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html
-        if: ${{ matrix.run_triton_tests }}
+          # Install torchax
```

Comment:
BTW this won't be needed once https://github.com/pytorch/xla/pull/8895 lands (soon)